index.php

```diff
@@ -11,9 +11,23 @@
 
 $collection = $sql->fetchAll();
 
+function makeCard($fish){
+    foreach( $fish as $fish) {
+        echo '<div class="fish-card">';
```

Comment:
does this have to be individual echo lines? could maybe make a variable to store this HTML block unit all at once?